dotCMS/src/main/java/com/dotmarketing/util/ImportStarterUtil.java

```diff
@@ -0,0 +1,1148 @@
+
+
+            if (_className.equals("Counter")) {
+                counter = true;
+            } else if (_className.equals("Image")) {
+                image = true;
+            } else if (_className.equals("Portlet")) {
+                portlet = true;
+            } else if (_className.equals("LogsMappers")) {
+                logsMappers = true;
+            } else {
+                try {
+                    _importClass = Class.forName(_className);
+                } catch (Exception e) {
+                    Logger.error(this, "Class not found " + _className);
+                    return;
+                }
+            }
+
+            Logger.info(this, "Importing:\t" + _className);
+
+
+            List l = new ArrayList();
+
+            List all = (List) BundlerUtil.xmlToObject(f);
+            if (filter != null) {
+                for (Object obj : all)
+                    if (filter.includeIt(obj))
+                        l.add(obj);
+            } else {
+                l = all;
+            }
+
+
+            Logger.info(this, "Found :\t" + l.size() + " " + _className + "(s)");
+            if (counter) {
+                for (int j = 0; j < l.size(); j++) {
+                    final HashMap<String, String> dcResults = (HashMap<String, String>) l.get(j);
+
+                    DotConnect dc = new DotConnect();
+                    dc.setSQL("insert into counter values (?,?)");
+                    dc.addParam(dcResults.get("name"));
+                    dc.addParam(Integer.valueOf(dcResults.get("currentid")));
+                    dc.loadResults();
+
+                }
+            } else if (image) {
+                /*
+                 * The changes in this part were made for Oracle databases. Oracle has problems when getString()
+                 * method is called on a LONG field on an Oracle database. Because of this, the object is loaded
+                 * from liferay and DotConnect is not used
+                 */
+                for (int j = 0; j < l.size(); j++) {
+                    final Image im = (Image) l.get(j);
+
+                    DotConnect dc = new DotConnect();
+                    dc.setSQL("insert into image values (?,?)");
+                    if (!UtilMethods.isSet(im.getImageId()) && DbConnectionFactory.isOracle()) {
+                        return;
+                    }
+                    dc.addParam(im.getImageId());
+                    dc.addParam(im.getText());
+                    dc.loadResults();
+
+                }
+            } else if (portlet) {
+                for (int j = 0; j < l.size(); j++) {
+                    final HashMap<String, String> dcResults = (HashMap<String, String>) l.get(j);
+
+                    DotConnect dc = new DotConnect();
+                    StringBuffer sb = new StringBuffer("insert into portlet values (?,?,?,?,");
+                    if (dcResults.get("narrow").equalsIgnoreCase("f") || dcResults.get("narrow").equalsIgnoreCase("false")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "narrow" appears 4 times in this file; the first occurrence is on line 560](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)